digits/standard-networks/torch/ImageNet-Training/alexnet.lua

```diff
@@ -80,11 +67,10 @@ return function(params)
         assert(params.inputShape[2]==256 and params.inputShape[3]==256, 'Network expects 256x256 images')
     end
     return {
-        model = createModel(1, channels),
+        model = createModel(params.ngpus, channels),
```

Comment:
Multi-GPU is still not supported, right? But I guess it's blocked in DIGITS before you ever get this far so this is ok?